recipes/vulkan-loader/all/conanfile.py

```diff
@@ -176,7 +179,7 @@ def package_info(self):
         self.cpp_info.libs = [f"vulkan{suffix}"]
 
         # allow to properly set Vulkan_INCLUDE_DIRS in CMake like generators
-        self.cpp_info.includedirs = self.dependencies["vulkan-headers"].cpp_info.aggregated_components().includedirs
+        self.cpp_info.includedirs = self.dependencies["vulkan-headers"].cpp_info.includedirs
```

Comment:
It might be necessary to define ``self.cpp_info.includedirs = []`` to invalidate the default ``include`` directory

Reply:
Why is that necessary? Is it due to caching somewhere?
Admittedly, I forgot to test this with `full_deploy`